src/pages/library/DataBrowserDetails.js

```diff
@@ -169,6 +171,8 @@ const Sidebar = ({ snapshot, id, setShowRequestAccessModal }) => {
       ])
     ]),
     h(ButtonPrimary, {
+      disabled: snapshot.access !== snapshotAccessTypes.GRANTED,
+      tooltip: snapshot.access === snapshotAccessTypes.GRANTED ? '' : 'You do not have access to this dataset. Please request access to unlock this feature.',
```

Comment:
Is there a way to share text to avoid duplication? (Or maybe this isn't something that we do in terra-ui right now?)

Reply:
We don't currently do that, but I can see the value... I'll add it to the databrowser-utils constants